### PR TITLE
Addition of 1 AHU, 2 DOAS and 1 MAU.

### PR DESCRIPTION
**Pull request by ghairapetian**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 1]
```diff
@@ -4844,4 +4844,24 @@ AHU_SFSS_SFMSC_SFVSC_FDPM_CHWSC_RTM_RACO2C_SSPC2X_VOADM2X_SAIDC2X_RAIDC2X_SSPC:
...
   - RAIDC2X
-  - SSPC
\ No newline at end of file
+  - SSPC
+
+AHU_CHWSC_EFSS_EFVSC_FDPM3X_HTWHLSTC_HWSC_RCO2M_RSPC_SFM_SFSS_SFVSC_SSPC_VOADM_VMADC:
+  description: "Complex multi zone AHU with heat recovery wheel and recirculated air control."
+  is_canonical: true
+  implements:
+  - DOAS
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4853]: DOAS or AHU?

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml [hunk 2]
```diff
@@ -660,6 +660,21 @@ MAU_SFVSC_CHWSC_SSPC_FDPSM2X:
...
+  implements:
+  - MAU
+  - CHWSC
+  - HWSC
+  - SSPC
+  - SFSS
+  - SFVSC
+  - FDPM2X
+  - SFM
+  - OA
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/MAU.yaml:676]: this is usually assigned to its own device (a weather station). is this really used by this device?

**Comment by tasodorff** [comment]
@Karthi8883 check the DOAS inherited by the AHU, and confirm if OA is needed for the MAU. otherwise good to go.

File: ontology/yaml/resources/HVAC/entity_types/MAU.yaml (hunk 2 above)
Comment (tasodorff) [reply ontology/yaml/resources/HVAC/entity_types/MAU.yaml:676]: bump on this.

**Review by tasodorff — COMMENTED** [review]
See comment on `OA` for `MAU.yaml`

**Review by tasodorff — APPROVED** [review]

**Comment by cstirdivant** [comment]
@ghairapetian please sync your branch with the google:master

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**